dev_tools/packaging/isolated_packages_test.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+from unittest import mock
+
+import pytest
+
+from dev_tools import shell_tools
+from dev_tools.modules import list_modules
+
+# point, that is then cloned to a separate folder for each test.
```

Comment:
was this line copied from somewhere else? why does it start with point?

Reply:
not sure - it is weird! :)